tests/Any/SignerTests.cpp

```diff
@@ -88,6 +103,36 @@ TEST(Signer, NetworkNotSupported) {
     ASSERT_EQ("Network not supported", output.error().description());
 }
 
+TEST(Signer, WanchainTransactionSign) {
+    auto transaction = R"({"chainId": "MQ==","toAddress": "0x3535353535353535353535353535353535353535","nonce": "OQ==","gasPrice": "MjAwMDAwMDAwMDA=","gasLimit": "MjEwMDA=","amount": "MTAwMDAwMDAwMDAwMDAwMDAwMA=="})";
```

Comment:
@hewigovens we need to find a solution for `bytes`, this interface does not look good `{"chainId": "MQ=="}`  and as the rest of the types

Reply:
I think base64 is designed by protobuf json format